cmd/createorupdate/createorupdate.go

```diff
@@ -23,6 +24,17 @@ import (
 	"github.com/openshift/openshift-azure/pkg/util/azureclient"
 )
 
+const (
+	// https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/application_list
+	// To list and patch AAD applications, this code needs to have the clientID
+	// of an application with the following permissions:
+	// API: Windows Azure Active Directory
+	//   Delegated permissions:
+	//      Sign in and read user profile
+	//      Access the directory as the signed-in user
+	clientID = "5935b8e2-3915-409c-bfb2-865b7a9291e0"
```

Comment:
Whose client id does this correspond?

Reply:
@kwoodson you were right ... today this came back and bit me ;-)